bql/semantic/semantic.go

```diff
@@ -122,6 +128,33 @@ type GraphClause struct {
 	OTemporal        bool
 }
 
+// ConstructClause represents a singular clause within a construct statement.
+type ConstructClause struct {
```

Comment:
Yep, it is just the information required to insert a new triple into the graph. For instance (a contrived example):

`construct {?s ?p ?o; ?p1 ?o1} into ?a from ?b where {?s ?p ?o. ?s ?p1 ?o2}`

will lead to the creation of the following instance of `constructClauses`:

```
[]*ConstructClauses{
  {
    SBinding: ?s,
    PBinding: ?p,
    OBinding: ?o,
    ReificationClauses: []*ReificationClause{
      {
         PBinding: ?p1
         OBinding: ?o1
      },
   },
}

```
I guess the name is confusing. Do you have any suggestions?